inlong-agent/pom.xml

```diff
@@ -36,4 +36,54 @@
         <module>agent-release</module>
         <module>agent-docker</module>
     </modules>
+
+    <properties>
+        <debezium.version>1.8.0.Final</debezium.version>
```

Comment:
it's better to add `<debezium.version>1.8.0.Final</debezium.version>` to root pom?

Reply:
The versions of module dependencies are inconsistent, and there is no way to unify them.